src/store/conversationsStore.js

```diff
@@ -209,7 +251,9 @@ const actions = {
 	 * @param {object} conversation the conversation;
 	 */
 	addConversation(context, conversation) {
-		context.dispatch('emitUserStatusUpdatedWhenChanged', conversation)
+		if (conversation.type === CONVERSATION.TYPE.ONE_TO_ONE) {
```

Comment:
```suggestion
		if (conversation.status) {
```

Group conversations don't have a `status` field, so I'm not sure if we needed that additional check before and need that now.

Reply:
@nickvergessen Do you remember if there was a reason to add this check here: https://github.com/nextcloud/spreed/pull/9411

Except readability

Reply:
It could be that the user has no status now, but had one before. Then the event should trigger and delete the status.
Can e.g. happen when they go invisible, or when the inactive status gets purged.

That being said, it doesn't make sense with the rest of the code, so I guess no.

Reply:
As we never have this request without `includeStatus`, I added emit for null status as emitting about status removal.